packages/dds/tree/src/simple-tree/api/tree.ts

```diff
@@ -396,47 +405,53 @@ export interface SchemaCompatibilityStatus {
 	 *
 	 * - schema repository `A` has extra schema which schema `B` doesn't have, but they are unused (i.e. not reachable from the root schema)
 	 *
-	 * - field in schema `A` has allowed field members which the corresponding field in schema `B` does not have, but those types are not constructible (ex: an object node type containing a required field with no allowed types)
+	 * - field in schema `A` has allowed field members which the corresponding field in schema `B` does not have, but those types are not constructible (for example: an object node type containing a required field with no allowed types)
 	 *
 	 * These cases are typically not interesting to applications.
+	 *
+	 * Note that other content in the stored schema that does not impact document compatibility, like {@link NodeSchemaOptionsAlpha.persistedMetadata}, does not affect this field.
+	 *
+	 * For the computation of this equivalence, {@link SchemaStaticsAlpha.staged | staged} schemas are not included.
+	 * If there are any unknown optional fields, even if allowed by {@link SchemaFactoryObjectOptions.allowUnknownOptionalFields}, `isEquivalent` will be false.
 	 */
 	readonly isEquivalent: boolean;
 
 	/**
 	 * Whether the current view schema is sufficiently compatible with the stored schema to allow viewing tree data.
 	 * If false, {@link TreeView.root} will throw upon access.
+	 * @remarks
+	 * Currently, this field is true if `isEquivalent` is true as well as the following case:
 	 *
-	 * Currently, this field is true iff `isEquivalent` is true.
-	 * Do not rely on this:
-	 * there are near-term plans to extend support for viewing documents when the stored schema contains additional optional fields not present in the view schema.
-	 * The other two types of backward-compatible changes (field relaxations and addition of allowed field types) will eventually be supported as well,
-	 * likely through out-of-schema content adapters that the application can provide alongside their view schema.
-	 *
-	 * Be aware that even with these SharedTree limitations fixed, application logic may not correctly tolerate the documents allowable by the stored schema!
-	 * Application authors are encouraged to read docs/user-facing/schema-evolution.md and choose a schema compatibility policy that
-	 * aligns with their application's needs.
+	 * If the view schema sets {@link SchemaFactoryObjectOptions.allowUnknownOptionalFields} to true, documents with additional optional fields in the corresponding schema will also be viewable.
```

Comment:
Nit: this might be a bit easier to parse if formatted like:
> Currently, this is true in the following cases:
> 
> - if `isEquivalent` is true
> 
> - ...

Reply:
Should be better now